packages/bind/spec/crossWindowBehaviors.ts

```diff
@@ -66,11 +65,11 @@ describe('Cross-window support', function () {
 
     win2.document.write(BLANK_HTML)
     win2.document.close()
-    body2 = win2.document.body
+    const body2 = win2.document.body
 
     // renderTemplate
     window.runs(function () {
-      setTemplateEngine(new dummyTemplateEngine({ someTemplate: "<div data-bind='text: text'></div>" }), true)
```

Comment:
unclear why we're removing `, true` (may see it later, just noting)

Reply:
dummyTemplateEngine has only one parameter, now we a type-safe(r) 👯‍♀️ 👯‍♀️ 
![grafik](https://github.com/user-attachments/assets/7e5e81be-d245-4cbe-8f41-22395a4adcbb)